cpp/daal/include/services/internal/sycl/math/mkl_blas.h

```diff
@@ -76,7 +79,7 @@ struct MKLGemm
         auto c_ptr = c_usm.get() + offsetC;
 
         status |= catchSyclExceptions([&]() mutable {
-            ::oneapi::fpk::blas::gemm(_queue, transamkl, transbmkl, m, n, k, alpha, a_ptr, lda, b_ptr, ldb, beta, c_ptr, ldc);
+            mkl::blas::gemm(_queue, transamkl, transbmkl, m, n, k, alpha, a_ptr, lda, b_ptr, ldb, beta, c_ptr, ldc);
```

Comment:
Yes, most of the changes are made because for the purpose of testing with big oneMKL as we do not yet have the respective functionality in microlibs.
The changes will be reverted when the microlibs are ready.